### PR TITLE
Add --[source|target]-registry flags to images push cmd

### PR DESCRIPTION
**Pull request by 0xFelix**
**What this PR does / why we need it**:

This allows to specify a source registry to check if updates are needed
and a target registry to push freshly built containerdisks to.
In general this makes specifying registries less ambiguous.

**Special notes for your reviewer**:

Merge after #21

**Release note**:

```release-note
Allow to specify a source registry to check if new containerdisks need to be built.
Allow to specify a target registry to push freshly built containerdisks to.
```

### CODE REVIEW

File: cmd/medius/images/push.go [hunk 1]
```diff
@@ -64,6 +68,7 @@ func NewPublishImagesCommand(options *common.Options) *cobra.Command {
 	}
 	publishCmd.Flags().BoolVar(&options.PublishImagesOptions.ForceBuild, "force", options.PublishImagesOptions.ForceBuild, "Force a rebuild and push")
 	publishCmd.Flags().IntVar(&options.PublishImagesOptions.Workers, "workers", options.PublishImagesOptions.Workers, "Number of parallel workers")
+	publishCmd.Flags().StringVar(&options.PublishImagesOptions.TargetRegistry, "target-registry", "", "Target registry to push images to")
```
Comment (rmohr) [inline cmd/medius/images/push.go:71]: @0xFelix could you add a little bit of context why the existing `--registry` can't be used for that?

Comment (rmohr) [reply cmd/medius/images/push.go:71]: If this is to separate `--source-registry` from `--target-registry` for different subcommands, how about renaming `registry`?

Comment (0xFelix) [reply cmd/medius/images/push.go:71]: I added this flag so it is possible to check the production registry if an update is necessary and to push it to an intermediate registry first (for verification purposes).

Comment (rmohr) [reply cmd/medius/images/push.go:71]: I would also be fine to remove the `--registry` flag from the base args and just add on-demand to each command a more specific flag which is less ambiguous in source/target scenarios.

Comment (0xFelix) [reply cmd/medius/images/push.go]: Done, can you review again?

Comment (rmohr) [reply cmd/medius/images/push.go]: @0xFelix can it be that you added more to this PR than the description says?

Comment (rmohr) [reply cmd/medius/images/push.go]: Oh, you want #21 first. Shouldn't it be the other way round?

**Comment by rmohr** [comment]
Looks good, just needs a rebase.

**Pull request merged**